src/ImageSharp/Processing/Transforms/TransformHelpers.cs

```diff
@@ -12,6 +14,38 @@ namespace SixLabors.ImageSharp
     /// </summary>
     internal class TransformHelpers
     {
+        /// <summary>
+        /// Updates the dimensional metadata of a transformed image
+        /// </summary>
+        /// <typeparam name="TPixel">The pixel format.</typeparam>
+        /// <param name="image">The image to update</param>
+        public static void UpdateDimensionalMetData<TPixel>(Image<TPixel> image)
```

Comment:
Looks like you are missing an `a` here.

Reply:
Where? Am I being blind?

Reply:
`UpdateDimensionalMetData`
`UpdateDimensionalMetaData`

Reply:
Dammit! :smile: